app/uk/gov/hmrc/agentassurance/repositories/PropertiesRepository.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentassurance.repositories
+
+import javax.inject.{Inject, Singleton}
+
+import play.api.libs.json.Format
+import play.api.libs.json.Json.format
+import play.modules.reactivemongo.ReactiveMongoComponent
+import uk.gov.hmrc.mongo.{AtomicUpdate, ReactiveRepository}
+import uk.gov.hmrc.agentassurance.model._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class PropertiesRepository @Inject()(mongoComponent: ReactiveMongoComponent)
```

Comment:
index by key would be welcome, we can also add `StrictlyEnsuredIndexes` trait from template